app/views/tag_wranglings/index.html.erb

```diff
@@ -1,23 +1,88 @@
 <div class="tag wrangling home">
   <!--Descriptive page name, messages and instructions-->
-  <h2 class="heading"><%= ts('Tag Wrangling') %></h2>
+  <h2 class="heading"><%= t(".heading") %></h2>
   <% unless params[:show] %>
-    <h4 class="landmark heading"><%= ts('Notes') %></h4>
+    <h4 class="landmark heading"><%= t(".notes.landmark_heading") %></h4>
     <ul class="notes">
-      <li>Make tags canonical according to the <%= link_to 'wrangling guidelines', wrangling_guidelines_path %>.</li>
-      <li>If you have trouble accessing the wiki, please <a href="mailto:tagwranglers-personnel@transformativeworks.org">email tag wranglers personnel</a>. For any other questions relating to tag wrangling, please <a href="mailto:tagwranglers-staff@transformativeworks.org">email wrangling staff</a>, or for confidential issues please <a href="mailto:tagwranglers-chair@transformativeworks.org">email tag wrangling chairs</a>.</li>
+      <li>
+        <%= t(".notes.wrangling_guidelines_html",
+              wrangling_guidelines_link: link_to(t(".notes.wrangling_guidelines"), wrangling_guidelines_path)) %>
```

Comment:
For the note section to be consistent within all three points, the variable for the overall string here should also be different from the link variable, so something like `wrangle_to_guidelines_html` (while keeping the link the same) would be better